app/public/css/main.css

```diff
@@ -0,0 +1,1734 @@
+/**
```

Comment:
oh! it's a leftover of a change that was reverted already. you can safely remove it